config.json

```diff
@@ -382,6 +384,8 @@
         "show_accept_paypal": false,
         "show_accept_visa": false,
         "show_product_details_tabs": true,
+        "show_product_weight": true,
+        "show_product_dimensions": true,
```

Comment:
Not needed. This only needs to be added once to the global settings on line 72 and 73

I noticed that there are several cases like this :(. It only needs to be added to the variations if the values are different.

Reply:
Removed and GTK!